test/scripts/start-services.sh

```diff
@@ -2,21 +2,47 @@
 set -eu
 
 # config directory
-configdir=$(mktemp -d -t artemis-config-XXX)
+rm -rf tmp
+mkdir tmp
+configdir=$(pwd)"/tmp"
+
+# kill all potentially old processes
+kill $(ps -aux | grep -e polkadot/target -e ganache-cli -e release/artemis | awk '{print $2}') || true
 
 start_ganache()
 {
     echo "Starting Ganache"
-    
+
     pushd ../ethereum
```

Comment:
This will use the wrong version of `ganache-cli`. See my comment in https://github.com/Snowfork/polkadot-ethereum/pull/355#discussion_r617755754.

Reply:
fixes